### PR TITLE
Eliminate package title

### PR DESCRIPTION
**Pull request by dannyjdev**
Package title was eliminated from
* DisplayPackage.cshtml
* _ListPackage.cshtml

Addresses https://github.com/NuGet/NuGetGallery/issues/7149

### CODE REVIEW

**Comment by shishirx34** [comment]
:shipit:

File: src/NuGetGallery/Views/Shared/_ListPackage.cshtml [hunk 1]
```diff
@@ -72,13 +72,6 @@
                 }
             </div>
 
-            @if (!StringComparer.OrdinalIgnoreCase.Equals(Model.Id, Model.Title))
-            {
-                <div class="package-title package-legacy-title">
```
Comment (xavierdecoster) [inline src/NuGetGallery/Views/Shared/_ListPackage.cshtml]: Are the `package-title` and `package-legacy-title` CSS styles still used?

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 2]
```diff
@@ -201,12 +201,6 @@
                     }
                 </h1>
 
-                @if (!Model.Deleted && !StringComparer.OrdinalIgnoreCase.Equals(Model.Id, Model.Title))
```
Comment (xavierdecoster) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Do we still need the `Title` property on the `Model`?

File: src/NuGetGallery/Views/Shared/_ListPackage.cshtml (hunk 1 above)
Comment (dannyjdev) [reply src/NuGetGallery/Views/Shared/_ListPackage.cshtml]: `package-title` is used in different elements as a header CSS class but `package-legacy-title` was just used on this element (Already removed).

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 2 above)
Comment (dannyjdev) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: We only showed the title on `DisplayPackage.cshtml` and `_ListPackage.cshtml`. I removed the Title in `PackageViewModel` (base model for those views).

**Pull request merged**